lib/rom/sql/types/pg.rb

```diff
@@ -5,21 +5,49 @@ module ROM
   module SQL
     module Types
       module PG
-        Sequel.extension(:pg_json)
+        # UUID
+
+        UUID = Types::Strict::String.constrained(format: Regexp.new('^([0-9a-f]{8})-([0-9a-f]{4})-([0-9a-f]{4})-([0-9a-f]{2})([0-9a-f]{2})-([0-9a-f]{12})$'))
```

Comment:
I wonder what SQL gets executed when you run `repository.find('foo')` with serial pks :) This should raise an error fwiw